themes/openy_themes/openy_rose/templates/page/page.html.twig

```diff
@@ -57,11 +57,12 @@
             {% endblock %}
           </div>
 
-          <a class="navbar-toggle visible-xs visible-sm" data-toggle="collapse" data-target=".sidebar-left" href="#">
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-          </a>
+          <button aria-controls="sidebar" aria-expanded="false" class="collapsed navbar-toggle visible-xs visible-sm" data-target="#sidebar" data-toggle="collapse" type=button>
+            <div class=sr-only>navigation menu</div>
```

Comment:
It should be translatable and start from capital letter `{{ 'Navigation menu'|t }}`.